lib/features/reps_events/reps_events.info

```diff
@@ -100,5 +100,5 @@ features[variable][] = scheduler_use_vertical_tabs_reps_event
 features[variable][] = xmlsitemap_settings_node_reps_event
 features[views_view][] = reps_events
 features_exclude[field_base][body] = body
-multisite_version = 2.5
-php = 5.6
+multisite_version = 2.6
```

Comment:
Can you double check that ? It's strange we have a diff on this if reps websites are already on 2.6.

Reply:
All reps are now in the new infra using php7.3 and NextEuropa platform 2.6